common/kvstore/src/main/java/org/apache/spark/util/kvstore/LevelDB.java

```diff
@@ -393,11 +393,6 @@ public static class TypeAliases {
     TypeAliases(Map<String, byte[]> aliases) {
       this.aliases = aliases;
     }
-
-    TypeAliases() {
-      this(null);
```

Comment:
It is a useless constructor, I think we can delete it. If we need to keep it, it may be better to change it to init with a empty map